ffi/android/build_rust.sh

```diff
@@ -8,6 +8,11 @@ cargo build --manifest-path $RUSTLIB/Cargo.toml
 # Update this line accordingly if you are not building *from* x86_64
 export PATH=$PATH:$NDK_HOME/toolchains/llvm/prebuilt/linux-x86_64/bin
 
+export CARGO_TARGET_AARCH64_LINUX_ANDROID_LINKER="$NDK_HOME/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android26-clang"
+export CARGO_TARGET_X86_64_LINUX_ANDROID_LINKER="$NDK_HOME/toolchains/llvm/prebuilt/linux-x86_64/bin/x86_64-linux-android26-clang"
+export CARGO_TARGET_ARMV7_LINUX_ANDROIDEABI_LINKER="$NDK_HOME/toolchains/llvm/prebuilt/linux-x86_64/bin/armv7a-linux-androideabi26-clang"
+export CARGO_TARGET_I686_LINUX_ANDROID_LINKER="$NDK_HOME/toolchains/llvm/prebuilt/linux-x86_64/bin/i686-linux-android26-clang"
+
```

Comment:
This looks good. Another idea for docker build: we can use `.env` file with all these paths. If it stored in the same directory as docker-compose docker will automatically apply those env vars. We will need docker-compose for that, but I think that's not problematic

Reply:
does this apply also outside docker? because these paths are also needed for local build. so if the answer is no, I would keep it as it is

Reply:
Does not apply, the idea to have both to simplify life of docker users :)

Reply:
I don't understand, life of docker users is already pretty simple (they have to run just 3 commands in order to do everything, and adding a `.env` file won't change that number), and thanks to this change also local users have an easier life (since now they don't have to write the config file for cargo, manually changing the `NDK_HOME`)